fury/tests/test_primitive.py

```diff
@@ -1,19 +1,29 @@
 import numpy as np
 import numpy.testing as npt
-import fury.primitive as fp
+import fury.primitive as fpz
 
 
 def test_vertices_primitives():
+    # Tests the default vertices of all the built in primitive shapes.
     l_primitives = [(fp.prim_square, (4, 3)),
-                    (fp.prim_box, (8, 3))]
+                    (fp.prim_box, (8, 3)),
```

Comment:
the test will fail if you do not update this too. so you need to add the `-0.5, 0.5, etc...`